canopy/app/src/App.js

```diff
@@ -61,19 +64,21 @@ function App() {
     })();
   }, []);
 
+  const userSaplingRoutes = userSaplingManifests.map(
+    ({ displayName, namespace, icon }) => {
+      return {
+        path: `/${namespace}`,
+        displayName,
+        logo: icon
+      };
+    }
+  );
+
   return (
-    <>
-      <nav>
-        {userSaplingManifests.map(({ displayName, namespace }) => {
-          return (
-            <a href={`/${namespace}`} key={namespace}>
-              {displayName}
-            </a>
-          );
-        })}
-      </nav>
-      <div ref={saplingDomNode} />
-    </>
+    <div className="app">
+      <SideNav userSaplingRoutes={userSaplingRoutes} />
+      <div className="view background-light" ref={saplingDomNode} />
```

Comment:
Maybe switch from using the background-light class to using the scss variable for consistency